Doc/library/asyncio-eventloop.rst

```diff
@@ -192,12 +192,24 @@ Running and stopping the loop
 .. coroutinemethod:: loop.shutdown_default_executor()
 
    Schedule the closure of the default executor and wait for it to join all of
-   the threads in the :class:`ThreadPoolExecutor`. After calling this method, a
-   :exc:`RuntimeError` will be raised if :meth:`loop.run_in_executor` is called
-   while using the default executor.
+   the threads in the :class:`~concurrent.futures.ThreadPoolExecutor`.
+   Once this method has been called,
+   using the default executor with :meth:`loop.run_in_executor`
+   will raise a :exc:`RuntimeError`.
 
-   Note that there is no need to call this function when
-   :func:`asyncio.run` is used.
+   The *timeout* parameter specifies the amount of time
```

Comment:
In 3.11 there is no timeout parameter (I can't find it in the source code).

Reply:
Indeed, good catch.

On looking again, hidden by GitHub's folding of unmodified code, the docs [have a `.. versionchanged:: 3.12`][ref] about the addition of `timeout`.

[ref]: https://github.com/python/cpython/pull/100270/files#diff-afc9fa8b85212e4ea584ce77305955d6600c58f10e0bf2c04e4153ee2d204652L205